src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -298,7 +298,7 @@ fieldset.schemaform-field-template {
   font-weight: bold;
   text-decoration: none;
   margin: 0.5em 0;
-  word-break: keep-all;
+  white-space: nowrap;
```

Comment:
The older `word-break: keep-all;` didn't actually do what it was supposed to do :-/